### PR TITLE
Seed a default switch to control `automountServiceAccountToken`

### PR DESCRIPTION
**Pull request by jcpunk**
**What this PR does / why we need it**:

A number of kubernetes best practices suggest setting `automountServiceAccountToken` explicitly.  This adds template stubs to simplify that process.

**Special notes for your reviewer**:

**If applicable**:
- [ ] this PR contains documentation
- [ ] this PR contains unit tests
- [ ] this PR has been tested for backwards compatibility

### CODE REVIEW

File: pkg/chartutil/create.go [hunk 1]
```diff
@@ -365,6 +367,7 @@ metadata:
   {{- with .Values.serviceAccount.annotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
+  automountServiceAccountToken: {{ .Values.serviceAccount.automount }}"
```
Comment (yxxhero) [inline pkg/chartutil/create.go:370]:
```suggestion
  automountServiceAccountToken: {{ .Values.serviceAccount.automount }}
```

**Comment by yxxhero** [comment]
@jcpunk did it test locally?

**Comment by jcpunk** [comment]
My attempt to build locally gave a bunch of go errors with the unmodified source and the modified source....

**Comment by yxxhero** [comment]
@jcpunk please show the errors.

**Comment by jcpunk** [comment]
After rebooting my host, the error is gone.  It builds and renders as I'd expect.

**Review by joejulian — APPROVED** [review]

**Pull request merged**